Gemfile

```diff
@@ -20,6 +20,7 @@ gem "sass-rails", "~> 5.0"
 gem "sdoc", "~> 0.4.0", group: :doc
 gem "uglifier", ">= 1.3.0"
 gem "unicorn"
+gem "reform"
```

Comment:
Not as a hard rule - for this case using reform removed boilerplate and allows all the validations to be declared on the form object, rather than in the models (thinking the uniqueness check esp)